idd/Energy+.idd.in

```diff
@@ -16134,6 +16134,29 @@ SurfaceProperty:HeatTransferAlgorithm:Construction,
        \type object-list
        \object-list ConstructionNames
 
+SurfaceProperty:HeatBalanceSourceTerm,
```

Comment:
Thanks for updating all this, it looks much better.  It looks like a tab character got put in the memo line below on accident.  I'd say hold off from fixing it until you know if there will be any other final changes.  I'll review it again asap.

Reply:
Okay. I found that I missed two more comments to address - adding output vars and add an equation to the engineering ref. I'll make another commit after your 2nd round review then.